docs/source/index.rst

```diff
@@ -20,6 +20,27 @@ information here will make it easier for newcomers to get started using
     All content of the docs is licensed under the MIT license. Especially for the examples
     you encounter: Feel free to use this code in your own projects!
 
+.. note::
+
+    As a quick reference, here are some often used modules, classes and methods:
+
+    Modules:
```

Comment:
This is just a matter of preference but I think the heading would look better in bold.